docs/06-web-development.Rmd

```diff
@@ -101,7 +101,70 @@ There are much better tutorials on HTML than this section, such as those offered
 
 ## CSS
 
-CSS is a lot of fun to play with, but it can also easily steal your time.
+The Cascading Stylesheets (CSS) language is used to describe the look and formatting of documents written in HTML or XML. CSS is responsible for the visual style of your site. CSS is a lot of fun to play with, but it can also easily steal your time.
```

Comment:
Actually this is the first time I have learned that CSS can be used to format XML. I just did a quick search and read this page: https://www.w3schools.com/xml/xml_display.asp

Sounds like we probably don't want to mention the possibility of XML here.